src/ja/oneoneoneraw/src/eu/kanade/tachiyomi/extension/ja/oneoneoneraw/OneOneOneRaw.kt

```diff
@@ -0,0 +1,215 @@
+package eu.kanade.tachiyomi.extension.ja.oneoneoneraw
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.decodeFromJsonElement
+import kotlinx.serialization.json.decodeFromStream
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class OneOneOneRaw : HttpSource() {
+
+    override val name = "111raw"
+
+    override val lang = "ja"
+
+    override val baseUrl = "https://111raw.com"
+
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimit(2)
+        .build()
+
+    private val apiUrl = "https://api.rawz.org/api/manga".toHttpUrl()
```

Comment:
Is it mirror/alt site to rawz perhaps?
https://github.com/keiyoushi/extensions-source/blob/5a1721eb8760d6137b62709c4e94e8a2a6dffd54/src/ja/rawz/src/eu/kanade/tachiyomi/extension/ja/rawz/RawZ.kt#L27

Reply:
Yup, appears to be a alt/scraper, so I don't see any point in having both as an extension. Could probably close #284.